src/components/Card/CardAvatar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import cardAvatarStyle from '../../assets/styles/components/cardAvatarStyle';
+
+function CardAvatar({ ...props }) {
+  const { classes, children, className, plain, profile, ...rest } = props;
+  const cardAvatarClasses = classNames({
+    [classes.cardAvatar]: true,
```

Comment:
I'm a bit confused on the purpose of the classes prop if they are being matched directly to another prop. This is   making a hard bind of dynamic property to another which makes this more brittle. I'd recommend changing this for an object that binds the classNames and the `plain` and `profile` props.

Reply:
Got it 👍